src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -249,10 +269,14 @@ public void returnsEmptyIteratorOnInvalidPartRequest() throws IOException {
                 )
             )
         );
-        MatcherAssert.assertThat(
-            multi.part("fake").iterator().hasNext(),
-            Matchers.is(false)
-        );
+        try {
+            MatcherAssert.assertThat(
+                multi.part("fake").iterator().hasNext(),
+                Matchers.is(false)
+            );
+        } finally {
```

Comment:
@raphaelln I'm not convinced about these finally blocks... Can't we just invoke `multi.body().close();` without finally?